packages/material-ui/src/Chip/Chip.js

```diff
@@ -457,7 +461,7 @@ Chip.propTypes = {
   /**
    * The color of the component. It supports those theme colors that make sense for this component.
    */
-  color: PropTypes.oneOf(['default', 'primary', 'secondary']),
+  color: PropTypes.oneOf(['default', 'inherit', 'primary', 'secondary']),
```

Comment:
Do we support color inherit? It seems that we don't.

Reply:
Fixed in #21375